util/string_utils.go

```diff
@@ -3,7 +3,7 @@ package util
 import "strings"
 
 func Capitalize(str string) string {
-	return strings.ToUpper(string(str[0])) + str[1:]
+	return strings.Title(strings.ToLower(str))
```

Comment:
This actually changes the behavior: "TEST" is now returned as "Test" and not "TEST". I am assuming this was the case, if not, I can change the patch to avoid touching other letters.

Reply:
I'm ok with the change, thanks for fixing it.